MatrixKit/Views/RoomBubbleList/MXKRoomBubbleTableViewCell.m

```diff
@@ -1337,23 +1346,36 @@ - (IBAction)onContentViewTap:(UITapGestureRecognizer*)sender
         }
         else if (self.messageTextView)
         {
-            // Check whether the user tapped in front of a text component.
-            tapPoint = [sender locationInView:self.messageTextView];
+            // NOTE: A tap on messageTextView using `MXKMessageTextView` class fallback here if the user does not tap on a link.
             
-            if (tapPoint.y > 0 && tapPoint.y < self.messageTextView.frame.size.height)
+            // Use the same hack as `onMessageTap:`, check whether the current displayed text corresponds to an attached file
+            // NOTE: This assume that the cell has only one `MXKRoomBubbleComponent`
```

Comment:
I would say: "This assumes that a cell with attachment has only one `MXKRoomBubbleComponent`"